pkg/config/piped.go

```diff
@@ -582,16 +582,16 @@ type Notifications struct {
 }
 
 type NotificationRoute struct {
-	Name         string   `json:"name"`
-	Receiver     string   `json:"receiver"`
-	Events       []string `json:"events"`
-	IgnoreEvents []string `json:"ignoreEvents"`
-	Groups       []string `json:"groups"`
-	IgnoreGroups []string `json:"ignoreGroups"`
-	Apps         []string `json:"apps"`
-	IgnoreApps   []string `json:"ignoreApps"`
-	Envs         []string `json:"envs"`
-	IgnoreEnvs   []string `json:"ignoreEnvs"`
```

Comment:
We should keep this to ensure that the old Piped configuration is still able to be loaded.

Reply:
That's true, thank you 🙏  https://github.com/pipe-cd/pipecd/pull/3241/commits/b422c497138d9c447cced7a45911ad25bf4ce3be